### PR TITLE
[1LP][RFR] fixing failing tests due to name change

### PR DESCRIPTION
**Pull request by vatsalparekh**
Purpose or Intent
=================
{{ pytest: -vvv cfme/tests/infrastructure/test_vm_reconfigure.py -k test_vm_reconfig_add_remove_disk_cold --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7cb0743d8f5d97287c5579ee913699fa755732b5:

`cfme/infrastructure/virtual_machines.py`:
- :bangbang: Line 284:63: **E203** *whitespace before ':'*
- :bangbang: Line 285:14: **E251** *unexpected spaces around keyword / parameter equals*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0664eab9c836e2c1a298fdaf26ba3d5dc155c9b4:

`widgetastic_manageiq/vm_reconfigure.py`:
- :bangbang: Line 35:18: **E251** *unexpected spaces around keyword / parameter equals*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: widgetastic_manageiq/vm_reconfigure.py [hunk 1]
```diff
@@ -30,7 +31,8 @@ class DisksTable(VanillaTable):
         # Workaround necessary until BZ 1524960 is resolved
         4: BootstrapSelect(id='hdUnit'),
         3: BootstrapSelect(id='hdUnit'),
-        'Dependent': BootstrapSwitch(name='cb_dependent'),
+        'Dependent': BootstrapSwitch(
```
Comment (izapolsk) [inline widgetastic_manageiq/vm_reconfigure.py]: @vatsalparekh, this issue should disappear when the following my PR gets merged https://github.com/RedHatQE/widgetastic.core/pull/68

**Comment by dajoRH** [comment]
Lint report for commit d53b0408d25404ce9b485e51dde135eca2a4e3c0:

`widgetastic_manageiq/vm_reconfigure.py`:
- :bangbang: Line 35:17: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 35:19: **E251** *unexpected spaces around keyword / parameter equals*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by vatsalparekh** [comment]
Requests are getting finished as `NoSuchElementException` is fixed here, getting `TimeOutError` unexpectedly at places

File: widgetastic_manageiq/vm_reconfigure.py (hunk 1 above)
Comment (vatsalparekh) [reply widgetastic_manageiq/vm_reconfigure.py]: You mean the issue that the `VersionPick` not getting resolved would disappear when that PR is merged, right?

**Comment by RonnyPfannschmidt** [comment]
looks good, i presume the current master depends on a wt version that supports this picking?

File: widgetastic_manageiq/vm_reconfigure.py (hunk 1 above)
Comment (izapolsk) [reply widgetastic_manageiq/vm_reconfigure.py]: exactly

**Pull request merged**